src/util/simplify_expr_floatbv.cpp

```diff
@@ -276,7 +277,18 @@ bool simplify_exprt::simplify_floatbv_op(exprt &expr)
 
   exprt op0=expr.op0();
   exprt op1=expr.op1();
-  exprt op2=expr.op2(); // rounding mode
+  exprt attached_rounding_mode = expr.op2(); // rounding mode
+  if(attached_rounding_mode.id() == ID_symbol)
+  {
+    const symbolt *p_rounding_sym;
+    if(
+      !ns.lookup(
+        to_symbol_expr(attached_rounding_mode).get_identifier(),
+        p_rounding_sym))
+    {
+      attached_rounding_mode = p_rounding_sym->value;
```

Comment:
How is using the initial value justified? It may, after all, change during program execution?